inc/targetbase.class.php

```diff
@@ -764,4 +773,76 @@ protected function parseTags($content, PluginFormcreatorForm_Answer $formanswer,
 
       return $content;
    }
+
+
+   protected function showLocationSettings($rand) {
+      global $DB;
+
+      echo '<tr class="line0">';
+      echo '<td width="15%">' . __('Location') . '</td>';
+      echo '<td width="45%">';
+      Dropdown::showFromArray('location_rule', static::getEnumLocationRule(), array(
+         'value'                 => $this->fields['location_rule'],
+         'on_change'             => 'change_location()',
+         'rand'                  => $rand
+      ));
+      $script = <<<EOS
```

Comment:
it's much a personal preference but for javascript heredoc syntax, i use JAVASCRIPT tag name, thereby my editor (sublime or vscode) highlight correctly the code